sfge/src/test/java/com/salesforce/TestUtil.java

```diff
@@ -131,7 +131,7 @@ public static void buildGraphForFls(
         buildGraphForFls(g, new String[] {sourceCode}, renderXml);
     }
 
-    public static void buildGraph(GraphTraversalSource g, String[] sourceCode) {
+    public static void buildGraph(GraphTraversalSource g, String... sourceCode) {
```

Comment:
Changing the signature slightly made the code easier to write, and it doesn't affect anything else since `String...` and `String[]` are automatically converted to each other.